client/src/components/Modal.jsx

```diff
@@ -83,58 +65,85 @@ const Right = styled(Arrow)`
   vertical-align: middle; 
   text-align: center; 
   padding-top: 280px;
-
 `;
 
 const Left = styled(Arrow)`
   display: table-cell; 
   vertical-align: middle; 
   text-align: center; 
   padding-top: 280px;
+`;
 
+const BottomBar = styled.div`
+  height: 123px;
+  width: 100%;
+`;
+
+const Caption = styled.div`
+  color: white;
+  font-size: 13px;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 200;
+  text-align: center;
+  padding-top: 50px;
+  vertical-align: middle;
+  line-spacing:
 `;
 
 class Modal extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      data: props.data,
+      currentPhoto: 0,
+    };
     this.closeModal = this.closeModal.bind(this);
+    this.previousImage = this.previousImage.bind(this);
+    this.nextImage = this.nextImage.bind(this);
   }
 
   closeModal(event) {
     this.props.closeModal && this.props.closeModal(event);
   }
 
+  previousImage() {
```

Comment:
Just being a little bit nit-picky, but since these are methods, it might be better to change the names of previousImage and nextImage to a verb (e.g. setToPreviousImage) to go along with the naming conventions.

Reply:
Thanks for the feedback! I agree with you. I'll change them to convey their functionality better.